steveg      03/08/15 06:13:44

Modified:    xinetd   nvlists.c
Log:
Add NULL entry to success_log_options to properly end the nvlist.

Revision  Changes    Path
1.4       +1 -0      xinetd/xinetd/nvlists.c

Index: nvlists.c
===================================================================
RCS file: /cvs/xinetd/xinetd/xinetd/nvlists.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- xinetd/xinetd/nvlists.c	23 Jun 2003 18:25:00 -0000	1.3
+++ xinetd/xinetd/nvlists.c	15 Aug 2003 13:13:44 -0000	1.4
@@ -70,6 +70,7 @@
       { "PID",         LO_PID      },
       { "USERID",      LO_USERID   },
       { "TRAFFIC",     LO_TRAFFIC  },
+      { CHAR_NULL,     0           }
    } ;
